docs/documentation/admin/workflows/admin_workflow_public_survey.md

```diff
@@ -0,0 +1,93 @@
+---
+id: admin_workflow_public_survey
+title: IV. Public Survey
+sidebar_label: IV. Public Survey
+---
+import useBaseUrl from '@docusaurus/useBaseUrl'; 
+import Highlight from '@theme/Highlight';
+
+<img alt="design" className="img_sizing" src={useBaseUrl('img/design/Workflows.svg')} />
+<br/>
+
+
+## Overview {#overview}
+Sometimes companies might want to give people outside the Cotalker environment the opportunity to create a task. For example, through a _shared link_, customers who recur for support can fill out a form with their issue and automatically create a Cotalker task that would act as a support ticket. We call this feature a _public survey_.
+
+
+## Basic Setup {#basic-setup}
+<div className="alert alert--secondary">
+
+1. From <span className="badge badge--primary">Administration</span>, go to <span className="badge badge--primary">Workflows</span> section, select a **workflow group**, and finally choose the corresponding **workflow**.
+
+  The [workflow settings panel](/docs/documentation/admin/workflows/admin_workflow_configure#edit-a-single-workflow) will open up.
+
+</div>
+<br/>
+
+<div className="alert alert--secondary">
+
+2. In the **Start Form** field, choose the desired survey. Once a _survey_ has been selected, the <span className="badge badge--primary">Share</span> button appears.
+
+<img alt="share survey" className="img_sizing item shadow--tl" src={useBaseUrl('img/admin_survey_share_00.png')} />
+<br/>
+
+</div>
+<br/>
+
+<div className="alert alert--secondary">
+
+3. Press the <span className="badge badge--primary">Share</span> button. A _Share survey_ dialog box will appear. The **Share** field has three options: _Restricted_, _Anybody with the Link_, and _Disabled_.
+
+<img alt="share dialog box" className="img_sizing item shadow--tl" src={useBaseUrl('img/admin_survey_share_01.png')} />
+<br/>
+
+:::note Sharing Options
+  - _Anyone with the Link_: Public link. It only works if there is no open Cotalker session in the browser. 
```

Comment:
Esto funciona si el usuario tiene sesión

Reply:
Para Anyone with the link y Restricted, si el usuario tiene sesión en una compañía diferente a la del formulario, esto no funciona